src/renderers/shaders/ShaderChunk/dithering_pars_fragment.glsl

```diff
@@ -0,0 +1,14 @@
+// based on https://www.shadertoy.com/view/MslGR8
+vec3 dithering( vec3 color ) {
+	//Calculate grid position
+	float grid_position = fract( dot( gl_FragCoord.xy - vec2( 0.5, 0.5 ) , vec2( 1.0 / 16.0, 10.0 / 36.0 ) + 0.25 ) );
+
+	//Shift the individual colors differently, thus making it even harder to see the dithering pattern
+	vec3 dither_shift_RGB = vec3( 0.25 / 255.0, -0.25 / 255.0, 0.25 / 255.0 );
```

Comment:
I'm generally fine with whatever changes you wanna make here as long as it looks good, but I did specifically select this algorithm because it only uses multiplication and addition. Let's keep it fast.